src/ValueObjects/TranslatedString.php

```diff
@@ -2,63 +2,50 @@
 
 namespace CultuurNet\SearchV3\ValueObjects;
 
+use JMS\Serializer\Annotation\HandlerCallback;
 use JMS\Serializer\DeserializationContext;
 use JMS\Serializer\JsonDeserializationVisitor;
-use JMS\Serializer\Annotation\HandlerCallback;
 
-/**
- * Provides a value object for translated strings like name and description.
- */
-class TranslatedString
+final class TranslatedString
 {
-
     /**
      * Translations
      *
-     * @var array
+     * @var string[]
      */
-    protected $values = [];
+    private $values = [];
```

Comment:
Gets set inside the constructor.

Reply:
Fixed in https://github.com/cultuurnet/search-v3/pull/43/commits/9268fd6f232389803cf606f39342d6fe054878a3